server/tests/article.test.js

```diff
@@ -0,0 +1,119 @@
+import 'babel-polyfill'
+import path from 'path'
+import { GraphQLError } from 'graphql'
+import { ApolloServer, gql } from 'apollo-server'
+import { describe, expect, jest, test } from '@jest/globals'
+import { importSchema } from 'graphql-import'
+import { createTestClient } from 'apollo-server-testing'
+import resolvers from '../resolvers'
+import { databaseApi, searchApi } from '../endpoints'
+import { articles } from './mocks'
+
+const typeDefs = importSchema(path.join(__dirname, '../typedefs/index.graphql'))
+const server = new ApolloServer({ typeDefs, resolvers })
+
+const tests = [
+  {
+    name: 'Normal request with a valid search',
+    query: gql`
+            query articles($search: String!) {
+                articles(search: $search) {
+                    id
+                    number
+                    content
+                    keywords
+                    minutesToRead
+                }
+            }
+        `,
+    variables: { search: 'Es obligatorio usar casco con bicicleta?' },
+    response: { data: { articles: ['1'] } },
+    expected: { articles: [articles[0]] }
```

Comment:
Is there a reason why response is 1 and expected is 0? Correct me if I'm wrong but is it because of the difference of the indexing of the mocked search articles v.s. yours?

Reply:
Yeah, `articles[0]` is just referencing the 1st article available in the test set, which is Article 1.